app/src/main/java/com/duckduckgo/app/bookmarks/db/BookmarksDao.kt

```diff
@@ -29,6 +29,9 @@ interface BookmarksDao {
     @Query("select * from bookmarks")
     fun bookmarks(): LiveData<List<BookmarkEntity>>
 
+    @Query("select * from bookmarks")
+    fun bookmarksSync(): List<BookmarkEntity>
```

Comment:
nit about naming: I get the `sync` means synchronous but it took me scanning thru the file to get that and only realised when I saw the other `bookmarks` method.
Could we call them `getBookmarks` and `getBookmarksSync` instead?

Reply:
I like that!